node/test/as-http.js

```diff
@@ -192,7 +192,7 @@ function allocHTTPBridge(opts) {
     }
 
     function onEgressRequest(hreq, hres) {
-        cluster.asHTTP.forwardToTChannel(cluster.egressChan, hreq, hres);
+        cluster.asHTTP.forwardToTChannel(cluster.egressChan, hreq, hres, {}, function nocallback(err){});
```

Comment:
isn't that a lint exception? (unused arg err?)  probably at least add `if (err) throw err;` to have it break the test